src/lightning/pytorch/loggers/neptune.py

```diff
@@ -223,8 +226,9 @@ def __init__(
         prefix: str = "training",
         **neptune_run_kwargs: Any,
     ):
-        if not _NEPTUNE_AVAILABLE and not _NEPTUNE_CLIENT_AVAILABLE:
-            raise ModuleNotFoundError(str(_NEPTUNE_AVAILABLE))
+        if not _NEPTUNE_AVAILABLE:
+            raise ModuleNotFoundError("Missing Neptune logger, please install it with `pip install neptune`")
```

Comment:
The previous `str(_NEPTUNE_AVAILABLE)` for printing was fine, no? We do it everywhere in Lightning

Reply:
```suggestion
            raise ModuleNotFoundError(str(_NEPTUNE_AVAILABLE))
```